src/storage/mvcc/reader/point_getter.rs

```diff
@@ -213,10 +218,14 @@ impl<S: Snapshot> PointGetter<S> {
                     }
                     match write.short_value {
                         Some(value) => {
+                            let current_key = self.write_cursor.key(&mut self.statistics.write);
```

Comment:
Here it cost a extra read, I am not sure if I sure pass down a flag `check_can_be_cached` so when the flag is false that I could avoid the extra read.